test/testsynccfapi.cpp

```diff
@@ -67,31 +67,31 @@ bool itemInstruction(const ItemCompletedSpy &spy, const QString &path, const Syn
 SyncJournalFileRecord dbRecord(FakeFolder &folder, const QString &path)
 {
     SyncJournalFileRecord record;
-    folder.syncJournal().getFileRecord(path, &record);
+    [[maybe_unused]] const auto result = folder.syncJournal().getFileRecord(path, &record);
```

Comment:
why not `QVERIFY` ?
apply also to other `[[maybe_unused]]

Reply:
@mgallien Then the compilation gets broken with this error:
`error C2561: 'dbRecord': function must return a value`

Reply:
OK